CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/plugins/DataAccessViewWelcomePlugin.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.welcome.plugins;
+
+import au.gov.asd.tac.constellation.plugins.PluginInfo;
+import au.gov.asd.tac.constellation.plugins.PluginType;
+import au.gov.asd.tac.constellation.views.dataaccess.panes.DataAccessViewTopComponent;
+import au.gov.asd.tac.constellation.views.welcome.WelcomePluginInterface;
+import au.gov.asd.tac.constellation.views.welcome.WelcomeTopComponent;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.VBox;
+import javax.swing.SwingUtilities;
+import org.openide.util.NbBundle;
+import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
+
+/**
+ * The Open Data Access View for the Welcome Page.
+ *
+ * @author Atlas139mkm
+ */
+@PluginInfo(pluginType = PluginType.CREATE, tags = {"WELCOME"})
+@NbBundle.Messages("DataAcessViewWelcomePluging=Data Acess View Welcome Plugin")
+public class DataAccessViewWelcomePlugin implements WelcomePluginInterface {
+
+    public static final String IMPORT = "resources/welcome_data_access.png";
+    private final ImageView importImage = new ImageView(new Image(WelcomeTopComponent.class.getResourceAsStream(IMPORT)));
+    private final Button importButton = new Button();
```

Comment:
Given the label being used later is 'Open' rather than 'Import', I wonder if it would make more sense to change these fields to `OPEN`, `openImage`, and `openButton`